ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/PullQueryExecutor.java

```diff
@@ -668,7 +668,7 @@ private static TableRowsEntity proxyTo(
   ) {
     final RestResponse<List<StreamedRow>> response = serviceContext
         .getKsqlClient()
-        .makeQueryRequest(owner.location(), statement.getStatementText());
+        .makeQueryRequest(owner.location(), statement.getMaskedStatementText());
```

Comment:
This should not be masked. It's sending an actual request to another node in order to be executed.